image/build.gradle

```diff
@@ -4,4 +4,6 @@ dependencies {
     kapt libs.glide_compiler
 
     api libs.rx_helpers
+    implementation ('com.github.bumptech.glide:okhttp3-integration:4.4.0@aar')
```

Comment:
Why aren't you using the same `glide_version` defined in the top-level build.gradle?

Reply:
Updated.